build_tools/docker/manylinux2014_x86_64-release/Dockerfile

```
@@ -50,3 +50,16 @@ ENV BAZEL_LINKLIBS "-lstdc++ -lm"
 # fetch/build/install exactly what is needed.
 COPY install_tbb_manylinux2014.sh /usr/local/bin
 RUN bash /usr/local/bin/install_tbb_manylinux2014.sh
+
+######## AMD ROCM #######
+# Though the versions say latest it is known at the time of Docker compose.
+# Currently it is ROCM v5.2.1 and AMDGPU 22.20
+ARG ROCM_VERSION=latest
+ARG AMDGPU_VERSION=latest
+
+# Install the ROCm rpms
+RUN yum clean all
+RUN echo -e "[ROCm]\nname=ROCm\nbaseurl=https://repo.radeon.com/rocm/yum/$ROCM_VERSION/main\nenabled=1\ngpgcheck=0" >> /etc/yum.repos.d/rocm.repo
+RUN echo -e "[amdgpu]\nname=amdgpu\nbaseurl=https://repo.radeon.com/amdgpu/$AMDGPU_VERSION/rhel/7.9/main/x86_64\nenabled=1\ngpgcheck=0" >> /etc/yum.repos.d/amdgpu.repo
+
+RUN yum install -y rocm-dev
```

Seems like this should all be one RUN line. There's no need to create separate layers for each of these commands

```suggestion
RUN yum clean all \
  && echo -e \
    "[ROCm]\nname=ROCm\nbaseurl=https://repo.radeon.com/rocm/yum/$ROCM_VERSION/main\nenabled=1\ngpgcheck=0" \
    >> /etc/yum.repos.d/rocm.repo \
  && echo -e \
    "[amdgpu]\nname=amdgpu\nbaseurl=https://repo.radeon.com/amdgpu/$AMDGPU_VERSION/rhel/7.9/main/x86_64\nenabled=1\ngpgcheck=0" \
    >> /etc/yum.repos.d/amdgpu.repo \
  && yum install -y rocm-dev
```

---

done